scripts/create_dataset.py

```diff
@@ -15,8 +15,6 @@
 def generate_MazeTokenizer(
     junk,
```

Comment:
What is going on with this junk parameter?

Reply:
Either `pool.imap` or `tdqm` adds a first parameter to the wrapped function that tracks something like iteration or thread number or something like that. We're not using it.